src/app/shared/dialogs/screen-arrange/screen-arrange-sidebar/screen-arrange-sidebar.component.html

```diff
@@ -37,67 +35,33 @@ <h3>Visible Media</h3>
 </div>
 
 <div class="sidebar__items">
-  <mat-card *ngFor="let visibleItem of visibleItems"
-            (click)="onSelectMedia($event, matRippleInstance, visibleItem)"
-            [class.selected]="currentSelectedClip?.clip?.id === visibleItem.clip.id"
-            matRipple [matRippleDisabled]="true"
-
-            #matRippleInstance="matRipple">
-    <!-- TODO Extract Item  MatCard to its own component -->
-    <mat-card-header>
-      <div mat-card-avatar>
+  <mat-selection-list [multiple]="false">
+    <mat-list-option *ngFor="let visibleItem of visibleItems"
+                     [selected]="currentSelectedClip === visibleItem"
+                     (click)="onSelectMedia($event, visibleItem)"
+                     (mousedown)="preventEvent($event)"
+    >
+      <div mat-list-avatar class="clip-avatar">
         <app-clip-type [type]="visibleItem.clip.type"
-                       iconSize="36px"
-                       mat-list-icon>
-
+                       iconSize="40px">
         </app-clip-type>
+
+        <mat-icon *ngIf="unsavedChangesIds.includes(visibleItem.clip.id)"
+                  class="clip-modified-icon"
+                  svgIcon="edit"
+                  color="primary"
+        ></mat-icon>
       </div>
-      <mat-card-title>{{visibleItem.clip.name}}</mat-card-title>
-      <mat-card-subtitle>
-        {{visibleItem.clipSetting.visibility | visibilityToString}}
-        / {{visibleItem.clipSetting.position | positionToString }}
-      </mat-card-subtitle>
-      <div style="width: 40px; height: 40px">
+      <div mat-line>{{visibleItem.clip.name}}</div>
+      <div mat-line>{{visibleItem.clipSetting.visibility | visibilityToString}}</div>
+
+      <div mat-list-icon class="card-settings-icon">
         <button (click)="openMediaSettingsDialog($event, visibleItem)"
                 mat-icon-button>
           <mat-icon svgIcon="settings"></mat-icon>
         </button>
       </div>
-    </mat-card-header>
-
-    <mat-card-content>
-      Size:
+    </mat-list-option>
+  </mat-selection-list>
 
-      <ng-container *ngIf="visibleItem.clipSetting.position === 0">
```

Comment:
could you re-add those values maybe next to the Actions panel ?

Reply:
Hmm.. this leads a row more in the action bar which itself is not a problem. But you need to fix your `moveable-control-box`, because it is not updating the overlay when the browser or content resizes. I've added a small fix so that the gewd control only visibly resizes once, but it's not reacting to resize changes aswell atm (which is easily fixable now with the `forceUpdate()`. But the bigger problem is the overlay.. Can you fix that or help me fix it? I'll push my progress in a minute

Reply:
Oh I can try/fine tune those issues later , there will be a bigger UI fix sweep in the next weeks so this will be tackled as well ^^

Reply:
Okay.. just contact me if I can help you :)

Reply:
Will do! :)